src/genomicsdb.pyx

```diff
@@ -38,6 +38,60 @@ def connect(workspace,
         List of attributes to be queried, by default None. All attributes will be queried if None.
     segment_size : int, optional
         Segment Size, by default None. Allow GenomicsDB to configure one.
+
+    Returns
+    -------
+    GenomicsDB
+        GenomicsDB instance ready for queries
+
+    Raises
+    ------
+    GenomicsDBException
+         On failure to connect to the native GenomicsDB library
+    """
+    try:
+        return _GenomicsDB(workspace=workspace,
+                           callset_mapping_file=callset_mapping_file,
+                           vid_mapping_file=vid_mapping_file,
+                           attributes=attributes,
+                           segment_size=segment_size)
+    except Exception as e:
+        print(e)
+        raise GenomicsDBException("Failed to connect to the native GenomicsDB library")
```

Comment:
If the `print(e)` is just for debugging we could also do `raise GenomicsDBException(...) from e` here to keep the stack trace.

Reply:
Is this acceptable `raise GenomicsDBException("Failed to connect to the native GenomicsDB library using protobuf", e)`? I don't see the original exception getting printed out when using `raise GenomicsDBException(...) from e`.

Reply:
That syntax also seems good to me. Hmm that's interesting, I wonder if the `raise ... from ...` syntax works differently in Cython for some reason?